src/TBuffer.h

```diff
@@ -149,9 +161,12 @@ class TBuffer
     TBuffer cut(QPoint&, QPoint&);
     void paste(QPoint&, TBuffer);
     void setBufferSize(int s, int batch);
-    static const QList<QString> getComputerEncodingNames() { return csmEncodingTable.keys(); };
-    static const QList<QString> getFriendlyEncodingNames();
-    static const QString& getComputerEncoding(const QString& encoding);
+    static const QList<QString> getComputerEncodingNames() { return csmEncodingsMap.keys(); }
+    // Found issues with returning a const QString reference from the global
+    // static table - so keep it simple and reliable and return by value which
```

Comment:
Read up on C++11 move semantics ([example](https://mbevin.wordpress.com/2012/11/20/move-semantics/)) as well.

Reply:
But this is not a move but a copy - we do not want to destroy the original... 😖